main/acle.md

```diff
@@ -4468,6 +4482,13 @@ Reads a 32-bit system register.
 
 Reads a 64-bit system register.
 
+``` c
+  __uint128_t __arm_rsr128(const char* special_register);
```

Comment:
`const char* special_register` -> `const char *special_register` for consistency

Reply:
Done.